app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -131,6 +150,22 @@ class GameActivity : AppCompatActivity(), GameView {
             }
             false
         }
+        muteButton = findViewById<ImageButton>(R.id.muteChannelButton)
+        muteButton.setOnClickListener {
+            voiceChannel.mute(muteButton)
+        }
+    }
+
+    private fun initVoiceChat(is_test_mode: Boolean, gameUid: Long) {
+
+
+        voiceChannel = VoiceIpEngineDecorator(this)
+
+        val userId = Random.nextInt(100000000)
```

Comment:
As discussed, this will be changed with the id of the user once we will have a generated a full id without character also for user signed in with google?

Reply:
Yes indeed, for now the Google unique id for users has letters in it, but our voiceChannel uid requires an integer in Agora. This wouldn't be a problem if we only needed to feed it to Agora, but we need the function to be two-way to then extract which user is the loudest in the room chat. -> This should be done next week.